include/boost/uuid/string_generator.hpp

```diff
@@ -122,23 +133,27 @@ struct string_generator {
         static char const*const digits_end = digits_begin + 22;
```

Comment:
In the get_value(char c) case it should be:
`sizeof(digits_begin)/sizeof(char)`
and in the get_value(wchar_t c) case it should be:
`sizeof(digits_begin)/sizeof(wchar_t)`

so that hardcoded numbers are not used.